fluxcd/datadog_checks/fluxcd/metrics.py

```diff
@@ -16,6 +16,7 @@
 assert sorted(FLUX_V1_METRICS) == list(FLUX_V1_METRICS)
 
 FLUX_V2_METRICS = {
+    "gotk_resource_info": "gotk.resource.info",
```

Comment:
Thanks so much! Do you mind adding this metric to the `metadata.csv`? It would be great if we could add the output to the test fixtures as well